_oasis

```diff
@@ -19,7 +19,7 @@ Flag cli
 
 Test test
   Run$:               flag(tests)
-  Command:            $test_mustache
+  Command:            $test_mustache && $spec_mustache
```

Comment:
Oh wait, yes you're right. I didn't realize these are two test commands.

But that means that spec_mustache should have its own Test section. I'm not sure if && desugars to bash, but it's to avoid it regardless.